administrative-sdk/speechChallenge.js

```diff
@@ -54,52 +54,16 @@ class SpeechChallenge {
   }
 
   /**
-   * Callback used by createSpeechChallenge.
-   *
-   * @callback Sdk~speechChallengeCreatedCallback
-   * @param {its.SpeechChallenge} challenge Updated speech challenge domain model instance.
-   */
-  speechChallengeCreatedCallback(challenge) {}
-
-  /**
-   * Error callback used by createSpeechChallenge.
-   *
-   * @callback Sdk~speechChallengeCreatedErrorCallback
-   * @param {object[]} errors Array of errors.
-   * @param {its.SpeechChallenge} challenge Speech challenge domain model instance with unapplied changes.
-   */
-  speechChallengeCreatedErrorCallback(errors, challenge) {}
-
-  /**
    * Create a speech challenge.
    *
    * @param {its.SpeechChallenge} challenge A speech challenge object.
    * @param {Sdk~speechChallengeCreatedCallback} [cb] The callback that handles the response.
    * @param {Sdk~speechChallengeCreatedErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.